x/liquidfarming/spec/02_state.md

```diff
@@ -29,15 +27,13 @@ const (
 
 type RewardsAuction struct {
 	Id                     uint64
-	PoolId                 uint64
-	SellingRewards         sdk.Coins
-	BiddingCoinDenom       string // pool coin denom
-	SellingReserverAddress string
+	PoolId                 uint64 // Corresponding pool id of the target liquid farm
+	BiddingCoinDenom       string // corresponding pool coin denom
 	PayingReserveAddress   string
 	StartTime              time.Time
 	EndTime                time.Time
 	Status                 AuctionStatus
-	WinnerBidId            uint64
+	WinnerBidId            uint64 // id of winning bid of the auction
```

Comment:
Since we decide to remove bid id, the spec is changed to include the following fields:

```go
Winner  string
Rewards sdk.Coins
```

Reply:
Reflected.